### PR TITLE
Refactor storybook templates

### PR DESCRIPTION
**Pull request by dcmouyard**
Review #545 first.

This PR:
* Refactors the Storybook templates to use global data
* Pulls the WYSIWYG component directly into the Article component instead of having to do that manually in the page demos
* Fixes broken page demo
* Removes the redundant article template, and uses the page template for the article page demo
* Update demo pages to have controls (only `show_admin_info` for now)

### CODE REVIEW

File: source/05-pages/article.stories.jsx [hunk 1]
```diff
@@ -49,20 +51,29 @@ const articleDemoContent = `
...
+  day: {
+    short: '9',
+  },
   author_name: 'William Goldman',
-  body: mainContent,
+  content: articleDemoContent,
 });
 
-const Article = () => <PageWrapper>{parse(articleContent)}</PageWrapper>;
+const Article = args => <PageWrapper>{parse(articleContent(args))}</PageWrapper>;
```
Comment (slucero) [inline source/05-pages/article.stories.jsx:74]: Codacy has a fix for the issue: [Function component is not a function expression](https://app.codacy.com/gh/forumone/gesso/pullRequest?prid=8642047)
```suggestion
var Article = function(args) {
```

File: source/05-pages/homepage.stories.jsx [hunk 2]
```diff
@@ -55,7 +59,9 @@ const homepageContent = twigTemplate({
   homepage_grid_title: 'You Don’t Vote For Kings',
 });
 
-const Homepage = () => <PageWrapper>{parse(homepageContent)}</PageWrapper>;
-Homepage.args = {};
+const Homepage = args => <PageWrapper>{parse(homepageContent(args))}</PageWrapper>;
```
Comment (slucero) [inline source/05-pages/homepage.stories.jsx:62]: Codacy has a fix for the issue: [Function component is not a function expression](https://app.codacy.com/gh/forumone/gesso/pullRequest?prid=8642047)
```suggestion
var Homepage = function(args) {
```

Comment (dcmouyard) [reply source/05-pages/homepage.stories.jsx:62]: We use that convention pretty often. I’ll tackle all of those cases next week.

Comment (kmonahan) [reply source/05-pages/homepage.stories.jsx:62]: I don't think Codacy's suggestion is right here. If we want to switch to a function declaration, it should be:
```
function Homepage(args) {
  return (<PageWrapper>{parse(homepageContent(args))}</PageWrapper>);
}
```
We definitely don't want to use `var` in place of `const`. Not sure why Codacy's suggesting that.

**Review by coreylafferty — APPROVED** [review]

**Pull request merged**